packages/modeling/CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# 2.0.0-alpha.0 (2020-08-13)
+
+
+### Bug Fixes
+
+* **booleans:** corrected fromFakePolygons to ignore 0 length sides ([#552](https://github.com/jscad/openjscad.org/issues/552)) ([a44650b](https://github.com/jscad/openjscad.org/commit/a44650b2ce471b44ae2a40f4debddbc47e8d8c34))
+* **booleans:** added dynamic EPS for 2d boolean operations ([#535](https://github.com/jscad/openjscad.org/issues/535)) ([510e909](https://github.com/jscad/openjscad.org/commit/510e90901e65c63204c19e7f2d146edcbf4944ad))
+* **colors:** colorize is now immutable & returns new geometries ([#566](https://github.com/jscad/openjscad.org/issues/566)) ([916824c](https://github.com/jscad/openjscad.org/commit/916824c63a0bf8896d1b6f85a82b129a013bec58))
+* **mat4:** Missing variable declaration. ([6c68e81](https://github.com/jscad/openjscad.org/commit/6c68e81e8456e4208a8fe038653c464aa49f7487))
+* **geometries:** corrected order of transforms in geom2, geom3, and path2 ([2cde371](https://github.com/jscad/openjscad.org/commit/2cde3718fbd55b85f81c7dd9756ded6d9fe3214e))
+* **transforms:** adjusted transforms to allow various lengths arrays as parameters
+* **geometries:** reworked toCompactBinary and fromCompactBinary functions
+* **primitives:** enhance primitives settings & defaults
+* **geometries:** added color support to to/fromCompactBinary functions
+* **maths:** updated vec2 and vec3 to use the same logic as glmatrix
+* **maths:** changed line2/mat4/vec2/vec3/vec4 to create arrays of Numbers
+* **measurements:** enhanced measurements to cache values as object attributes
+* **utils:** exposed additional user utility functions
+* **geometries:** corrected order of transforms in geom2, geom3, and path2
+
+
+### Features
```

Comment:
I guess I should add the curves stuff and the bugfixes to this as I commit them

Reply:
curves is mentioned in the Features, although not very apparent.

FYI...
- each Fix requires a patch bump in version... 2.0.0 to 2.0.1
- each Feature requires a minor bump in version... 2.0.0 to 2.1.0